### PR TITLE
chore: update to cleanup and consolidate configurations

### PR DESCRIPTION
**Pull request by awentzel**
# Description
Updates code owner files and other root dotfiles to cleanup deprecated or never used artifacts. 

## Motivation & context
As preparatory work for release, I noticed some configurations that were not accurate or never used and this aims to remove this clutter for better maintainability as newcomers enter the product.

## Issue type checklist

- [x] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [x] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -56,6 +54,7 @@ build/ @janechu @nicholasrice @chrisdholt @awentzel
 /packages/utilities/fast-animation/ @nicholasrice @chrisdholt
 /packages/utilities/fast-colors/ @nicholasrice @chrisdholt
 /packages/utilities/fast-eslint-rules/ @nicholasrice @janechu @chrisdholt
+/packages/utilities/fast-tslint-rules/ @nicholasrice @janechu @chrisdholt
```
Comment (janechu) [inline .github/CODEOWNERS:57]: This was purposely left out as this package will be removed in the coming weeks

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -9,14 +9,12 @@
...
-*.md    @awentzel @chrisdholt @falkicon
+*.md @awentzel @chrisdholt @falkicon
 /docs/ @awentzel @chrisdholt @falkicon
 
 # Github management
 .github/ @awentzel @janechu @chrisdholt @nicholasrice
-
-# Circle Ci
-.circleci/ @awentzel @janechu @nicholasrice @chrisdholt
+.github/workflows/ @awentzel
```
Comment (janechu) [inline .github/CODEOWNERS:17]: Can you add one more person to this path? We typically aim to have a minimum of 2 users assigned to any path.

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -9,14 +9,12 @@
 
 # File type specific owners
 # Markdown specific files
-*.md    @awentzel @chrisdholt @falkicon
+*.md @awentzel @chrisdholt @falkicon
```
Comment (janechu) [inline .github/CODEOWNERS:12]: It looks like this indenting was originally intended to match with the indenting on line 8, can you also adjust the indenting on line 8?

Comment (awentzel) [reply .github/CODEOWNERS:12]: Your suggestion for consistency was good, however, I instead used one space after each path, which was more consistent with the majority of the rules in the file.

File: .github/CODEOWNERS (hunk 1 above)
Comment (awentzel) [reply .github/CODEOWNERS]: Thanks.  Removed

**Pull request merged**